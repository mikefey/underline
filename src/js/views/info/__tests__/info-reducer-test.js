import test from 'ava';
import {
  SHOW_INFO,
  HIDE_INFO,
} from './../info-constants';
import infoReducer from './../info-reducer.js';

test('default state', (t) => {
  t.snapshot(infoReducer(undefined, {}));
});

test('handles SHOW_INFO', (t) => {
  t.snapshot(infoReducer(undefined, { type: SHOW_INFO }));
});

test('handles HIDE_INFO', (t) => {
  t.snapshot(infoReducer(undefined, { type: HIDE_INFO }));
});
