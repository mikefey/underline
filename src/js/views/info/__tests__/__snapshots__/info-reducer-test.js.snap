// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default state 1`] = `
Object {
  "show": false,
}
`;

exports[`handles HIDE_INFO 1`] = `
Object {
  "show": false,
}
`;

exports[`handles SHOW_INFO 1`] = `
Object {
  "show": true,
}
`;
